### ISSUE TITLE
Chromium Issue while loading a page 

### ISSUE THREAD

**Issue by Sonu5555** [microsoft/playwright#6137]
@pavelfeldman I am sorry for the late reply, below is the issue while i trying to clicking sign up after loading the page
#5916 
Linking issue #5916 

Playwright Version: [1.9.2
]

Operating System: [e.g. Windows, Linux or Mac]
Node.js version: [v14.15.4]
Browser: [Chromium]
Extra: [any specific details about your environment]

----------------------------
CODE
```
const { chromium } = require('playwright');
(async () => {
  const browser = await chromium.launch({
    headless: false
  });
  const context = await browser.newContext();
  // Open new page
  const page = await context.newPage();
  // Go to https://letcode.in/
  await page.goto('https://letcode.in/');
  // Click a:has-text("Sign up")
  await page.click('a:has-text("Sign up")');
  // assert.equal(page.url(), 'https://letcode.in/signup');
  // ---------------------
  await context.close();
  await browser.close();
})();
```

---------------------------
```
PS C:\Users\Sonu\Desktop\test> set DEBUG=pw:browser*            
PS C:\Users\Sonu\Desktop\test> node chrome.js       
(node:11108) UnhandledPromiseRejectionWarning: page.goto: Navigation failed because page crashed!
=========================== logs ===========================
navigating to "https://letcode.in/", waiting until "load"
============================================================
Note: use DEBUG=pw:api environment variable to capture Playwright logs.Error
    at Object.captureStackTrace (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\utils\stackTrace.js:48:19)       
    at Connection.sendMessageToServer (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\connection.js:69:48)
    at Proxy.<anonymous> (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\channelOwner.js:64:61)
    at C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\frame.js:102:65
    at Frame._wrapApiCall (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\channelOwner.js:77:34)
    at Frame.goto (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\frame.js:100:21)
    at C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\page.js:296:60
    at Page._attributeToPage (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\page.js:231:20)
    at Page.goto (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\page.js:296:21)
    at C:\Users\Sonu\Desktop\test\chrome.js:10:14
(Use `node --trace-warnings ...` to show where the warning was created)
(node:11108) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:11108) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by JoelEinbinder** [comment]
It looks the environment variable `DEBUG` isn't being set correctly. I always forget how environment variables work in powershell. Maybe try it without the `*`? `set DEBUG=pw:browser`? You should see a bunch of logs starting with `pw:browser`.

**Comment by Sonu5555** [comment]
@JoelEinbinder 

PS C:\Users\Sonu\Desktop\test> set DEBUG=pw:browser
PS C:\Users\Sonu\Desktop\test> node chrome.js      
(node:15872) UnhandledPromiseRejectionWarning: page.goto: Navigation failed because page crashed!
=========================== logs ===========================
navigating to "https://letcode.in/", waiting until "load"
============================================================
Note: use DEBUG=pw:api environment variable to capture Playwright logs.Error
    at Object.captureStackTrace (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\utils\stackTrace.js:48:19)       
    at Connection.sendMessageToServer (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\connection.js:69:48)
    at Proxy.<anonymous> (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\channelOwner.js:64:61)
    at C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\frame.js:102:65
    at Frame._wrapApiCall (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\channelOwner.js:77:34)
    at Frame.goto (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\frame.js:100:21)
    at C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\page.js:296:60
    at Page._attributeToPage (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\page.js:231:20)
    at Page.goto (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\page.js:296:21)
    at C:\Users\Sonu\Desktop\test\chrome.js:12:14
(Use `node --trace-warnings ...` to show where the warning was created)
(node:15872) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of 
an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:15872) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that 
are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by JoelEinbinder** [comment]
Try it l like this:
```
$env:DEBUG="pw:browser"
node chrome.js
```

**Comment by Sonu5555** [comment]
@JoelEinbinder I think your magic spell worked. Please see the below 


PS C:\Users\Sonu\Desktop\test> $env:DEBUG="pw:browser"
PS C:\Users\Sonu\Desktop\test> node chrome.js
  pw:browser <launching> C:\Users\Sonu\AppData\Local\ms-playwright\chromium-854489\chrome-win\chrome.exe --disable-background-networking --enable-features=NetworkService,NetworkServiceInProcess --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-breakpad --disable-client-side-phishing-detection --disable-component-extensions-with-background-pages --disable-default-apps --disable-dev-shm-usage --disable-extensions --disable-features=TranslateUI,BlinkGenPropertyTrees,ImprovedCookieControls,SameSiteByDefaultCookies,LazyFrameLoading --disable-hang-monitor --disable-ipc-flooding-protection --disable-popup-blocking --disable-prompt-on-repost --disable-renderer-backgrounding --disable-sync --force-color-profile=srgb --metrics-recording-only --no-first-run --enable-automation --password-store=basic --use-mock-keychain --user-data-dir=C:\Users\Sonu\AppData\Local\Temp\playwright_chromiumdev_profile-9RmQUG --remote-debugging-pipe --no-sandbox --no-startup-window +0ms
  pw:browser <launched> pid=7256 +3ms
(node:1612) UnhandledPromiseRejectionWarning: page.goto: Navigation failed because page crashed!
=========================== logs ===========================
navigating to "https://letcode.in/", waiting until "load"
============================================================
Note: use DEBUG=pw:api environment variable to capture Playwright logs.Error
    at Object.captureStackTrace (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\utils\stackTrace.js:48:19)
    at Connection.sendMessageToServer (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\connection.js:69:48)      
    at Proxy.<anonymous> (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\channelOwner.js:64:61)
    at C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\frame.js:102:65
    at Frame._wrapApiCall (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\channelOwner.js:77:34)
    at Frame.goto (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\frame.js:100:21)
    at C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\page.js:296:60
    at Page._attributeToPage (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\page.js:231:20)
    at Page.goto (C:\Users\Sonu\Desktop\test\node_modules\playwright\lib\client\page.js:296:21)
    at C:\Users\Sonu\Desktop\test\chrome.js:12:14
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1612) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1612) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
  pw:browser [err] [7256:15504:0411/003322.838:ERROR:device_event_log_impl.cc(214)] [00:33:22.838] USB: usb_service_win.cc:389 
Could not read device interface GUIDs: The system cannot find the file specified. (0x2) +12s
  pw:browser [err] [7256:15504:0411/003322.844:ERROR:device_event_log_impl.cc(214)] [00:33:22.839] USB: usb_device_handle_win.cc:1054 Failed to read descriptor from node connection: A device attached to the system is not functioning. (0x1F) +3ms

**Comment by zlk89** [comment]
I saw the same issue when I was using old version of playwright, it appears that version will crash if you are running tests in parallel (creating/closing pages/context in parallel)

**Comment by dgozman** [comment]
@Sonu5555 Unfortunately, the logs don't tell much. Perhaps, you could try running `C:\Users\Sonu\AppData\Local\ms-playwright\chromium-854489\chrome-win\chrome.exe` manually from the terminal, and typing the url yourself? I hope that it will crash in the same way. If so, you might get a crash stack in the terminal to paste here.

**Comment by Sonu5555** [comment]
@dgozman The browser has opened with the expected url. Below is the command used
C:\Users\Sonu\AppData\Local\ms-playwright\chromium-854489\chrome-win\chrome.exe https://letcode.in/

But how can i do the click function from the above code   **await page.click('a:has-text("Sign up")');**

**Comment by dgozman** [comment]
> But how can i do the click function from the above code **await page.click('a:has-text("Sign up")');**

The `Navigation failed because page crashed` error means that the page process crashed. We tried running the script on our machine, but it does not crash, so the problem is most likely related to your setup. If you can trigger the crash (by launching the browser and clicking around manually) and give us a dump or stack trace, we'll be able to start looking at the cause. Otherwise, we don't have enough information to proceed.

To answer your question, just click normally with a mouse 😄 

If clicking around does not trigger the crash, try launching similarly to Playwright (copied the command line from the <launching> log line above except for "--remote-debugging-pipe") and click more:
```sh
C:\Users\Sonu\AppData\Local\ms-playwright\chromium-854489\chrome-win\chrome.exe --disable-background-networking --enable-features=NetworkService,NetworkServiceInProcess --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-breakpad --disable-client-side-phishing-detection --disable-component-extensions-with-background-pages --disable-default-apps --disable-dev-shm-usage --disable-extensions --disable-features=TranslateUI,BlinkGenPropertyTrees,ImprovedCookieControls,SameSiteByDefaultCookies,LazyFrameLoading --disable-hang-monitor --disable-ipc-flooding-protection --disable-popup-blocking --disable-prompt-on-repost --disable-renderer-backgrounding --disable-sync --force-color-profile=srgb --metrics-recording-only --no-first-run --enable-automation --password-store=basic --use-mock-keychain --user-data-dir=C:\Users\Sonu\AppData\Local\Temp\playwright_chromiumdev_profile-123456 --no-sandbox
```

**Comment by Sonu5555** [comment]
https://user-images.githubusercontent.com/76145288/115346295-a1367380-a1cd-11eb-8559-ce29613728ca.mp4

@dgozman attached the video recording, no error while trying navigating manually

**Comment by aslushnikov** [comment]
Unfortunately we can't act upon this unless we can repro locally. I have to close this.

@Sonu5555 if there's a script that we can run locally to repro this, please feel free to file a new issue!
